code/modules/hydroponics/grown/apple.dm

```diff
@@ -26,6 +26,26 @@
 	foodtype = FRUIT
 	juice_results = list("applejuice" = 0)
 	tastes = list("apple" = 1)
+	
+/obj/item/reagent_containers/food/snacks/grown/apple/poisoned
+	bitesize = 0.01
+	var/list/eaters = list()
+	
+/obj/item/reagent_containers/food/snacks/grown/apple/poisoned/on_bite(mob/living/carbon/biter)
+	..()
+	var/bite_times = eaters[biter]
+	if(!bite_times)
+		bite_times = 1
+	switch(bite_times)
+		if(1)
+			B.say("Haugh!")
+		if(2)
+			biter.say("Aurl!")
+		if(3)
+			biter.say("Hurgh!")
+			biter.die()
+	eaters[biter] = bite_times
```

Comment:
i uh, don't see a method of increasing bite_times

Reply:
Have you heard of the High Elves?

Reply:
WHY   WONT   YOU   DIE?